watertap/unit_models/gac.py

```diff
@@ -270,475 +299,394 @@ def build(self):
             units=pyunits.dimensionless,
-            doc="Equilibrium concentration of adsorbed phase with liquid phase",
+            doc="equilibrium concentration of adsorbed phase with liquid phase",
         )
-
-        self.mass_adsorbed = Var(
+        self.dg = Var(
             initialize=1e5,
-            bounds=(1e-8, None),
+            bounds=(0, None),
             domain=NonNegativeReals,
-            units=units_meta("mass"),
-            doc="Mass of contaminant absorbed at the time of replacement",
+            units=pyunits.dimensionless,
+            doc="solute distribution parameter",
         )
-
-        self.mass_adsorbed_saturated = Var(
-            initialize=1e5,
-            bounds=(1e-8, None),
+        self.N_Bi = Var(
+            initialize=10,
+            bounds=(0, None),
             domain=NonNegativeReals,
-            units=units_meta("mass"),
-            doc="Mass of contaminant adsorbed if fully saturated",
+            units=pyunits.dimensionless,
+            doc="Biot number",
         )
 
         # ---------------------------------------------------------------------
-        # gac bed properties
+        # bed dimensions and related variables
+
+        self.velocity_sup = Var(
+            initialize=0.001,
+            bounds=(0, None),
+            domain=NonNegativeReals,
+            units=units_meta("length") * units_meta("time") ** -1,
+            doc="superficial velocity",
+        )
+        self.velocity_int = Var(
+            initialize=0.002,
+            bounds=(0, None),
+            domain=NonNegativeReals,
+            units=units_meta("length") * units_meta("time") ** -1,
+            doc="interstitial velocity",
+        )
         self.bed_voidage = Var(
             initialize=0.5,
             bounds=(0, 1),
             domain=NonNegativeReals,
             units=pyunits.dimensionless,
-            doc="Adsorber bed void fraction",
-        )
-
-        self.bed_volume = Var(
-            initialize=100,
-            bounds=(1e-3, None),
-            domain=NonNegativeReals,
-            units=units_meta("length") ** 3,
-            doc="Adsorber bed volume",
+            doc="bed void fraction",
         )
-
-        self.bed_area = Var(
-            initialize=1,
+        self.bed_length = Var(
+            initialize=5,
             bounds=(0, None),
             domain=NonNegativeReals,
-            units=units_meta("length") ** 2,
-            doc="Adsorber bed area, single adsorber area or sum of areas for adsorbers in parallel",
+            units=units_meta("length"),
+            doc="bed length",
         )
-
         self.bed_diameter = Var(
             initialize=1,
             bounds=(0, None),
             domain=NonNegativeReals,
             units=units_meta("length"),
-            doc="Adsorber bed diameter, valid if considering only a single adsorber",
+            doc="bed diameter, valid if considering only a single adsorber",
```

Comment:
And for adsorbers in parallel, assuming each has this diameter?

Reply:
This is just for ease of fixing diameter as a variable when matching data to a single column's performance just because bed length and diameter are typically given, used in _most_ validation. This clarifier is here just because if considering multiple in parallel, the **volume** variable equals the sum of the volumes for the beds operating in parallel, so diameter then isn't directly relevant. This diameter variable isn't touched with respect to any dimensionless parameters or performance, just for the single column dimension insight. This reasoning is touched on in the documentation as well.